src/main/java/Duke.java

```diff
@@ -1,10 +1,424 @@
+import java.util.Scanner;
```

Comment:
You may want to extract out each command as a class to follow OOP principles and improve code quality